components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-local-service-provider.jsp

```diff
@@ -311,12 +312,12 @@
 							      
 							      	       <tr>
 							      	      	   <td>
-							      	      		<input name="provisioning_idp" id="" type="hidden" value="<%=idp.getIdentityProviderName()%>" />
-							      	      			<%=idp.getIdentityProviderName() + (idpStatus.get(idp.getIdentityProviderName()) != null && idpStatus.get(idp.getIdentityProviderName()) ? "" : disbleText)%>
+							      	      		<input name="provisioning_idp" id="" type="hidden" value="<%=Encode.forHtmlAttribute(idp.getIdentityProviderName())%>" />
+							      	      			<%=Encode.forHtmlContent(idp.getIdentityProviderName()) + idpStatus.get(Encode.forHtmlContent(idp.getIdentityProviderName())) != null && idpStatus.get(Encode.forHtmlContent(idp.getIdentityProviderName())) ? "" : disbleText%>
 							      	      		</td>
 							      	      		<td> 
 							      	      			<% if(selectedProIdpConnectors.get(idp.getIdentityProviderName()) != null) { %>
-							      	      				<select name="provisioning_con_idp_<%=idp.getIdentityProviderName()%>" style="float: left; min-width: 150px;font-size:13px;"><%=selectedProIdpConnectors.get(idp.getIdentityProviderName())%></select>
+							      	      				<select name="provisioning_con_idp_<%=idp.getIdentityProviderName()%>" style="float: left; min-width: 150px;font-size:13px;"><%=Encode.forHtmlContent(selectedProIdpConnectors.get(idp.getIdentityProviderName()))%></select>
```

Comment:
Encode value of attribute name as below.
provisioning_con_idp_<%=Encode.forHtmlAttribute(idp.getIdentityProviderName())%>